server/server.go

```diff
@@ -415,10 +414,13 @@ func New(
 	s.restore(ctx)
 	s.cleanupSandboxesOnShutdown(ctx)
 
+	var bindAddressStr string
 	bindAddress := net.ParseIP(config.StreamAddress)
 	if bindAddress == nil {
-		hostIPs := s.getHostIPs(config.HostIP)
-		bindAddress = hostIPs[0]
+		// If bind address isn't valid or unset, then listen on all IP addresses
+		bindAddressStr = ""
```

Comment:
do we need this case? can't we just check if bindAddress != nil, and if so set bindAddressStr? `var bindAddressStr string` sets it as ""

Reply:
oops, yeah. we can do that in a follow cleanup PR.

Reply:
fixing it here https://github.com/cri-o/cri-o/pull/3153